tests/v2/test_waku_rln_relay.nim

```diff
@@ -236,18 +254,20 @@ suite "Waku rln relay":
 
     # ctx holds the information that is going to be used for  the key generation
     var 
-      obj = RLNBn256()
+      obj = RLN[Bn256]()
       objPtr = unsafeAddr(obj)
-      ctx = objPtr
-    let res = newCircuitFromParams(merkleDepth, unsafeAddr parametersBuffer, ctx)
+      objptrptr = unsafeAddr(objPtr)
+      ctx = objptrptr
+    let res = new_circuit_from_params(merkleDepth, unsafeAddr parametersBuffer, ctx)
```

Comment:
How come this is an unsafeAddr? If it is part of interacting with Rust code, perhaps this can be encapsulated slightly better? Ditto other places

Reply:
Thanks for the comment, figured out that `addr` can also work fine as long as variables are not defined with `let`. So, I changed all `unsafeAddr` to `addr`